res/battle/graphic/meson.build

```diff
@@ -0,0 +1 @@
+subdir('pl_batt_obj')
```

Comment:
**suggestion**: Can we name the internal folder something more meaningful? Maybe `res/battle/graphics/objects`? We generally try to avoid using the actual NARC names outside of where strictly necessary for matching (e.g., on NARC names)

Reply:
I've renamed the directory, I considered renaming some of the other files (`pl_batt_obj.ignore`, `pl_batt_obj.order`) but these are more directly related to the final NARC name so I've left them as-is